dali/operators/math/normalize/normalize_utils.h

```diff
@@ -105,45 +112,93 @@ void CalcInvStdDev(const TensorListView<StorageCPU, float> &inv,
  *
  * @remarks The scaling is split into these two values for precision.
  */
-static void ScaleRSqrtKeepZero(float *data, int64_t n, float rdiv, float mul) {
-  int i = 0;
+static void ScaleRSqrtKeepZero(float *data, int64_t n, float eps, float rdiv, float mul) {
+  int64_t i = 0;
 
 #ifdef __SSE__
-  // vectorized version of the loop below
+  // Vectorized version of the loop below
+
+  // We calculate the following:
+  // mul * rsqrt(data[i] + eps)
+  //
+  // rsqrt needs an extra step of Newton-Raphson refinement:
+  // rough = approx_rsqrt(x)
+  // precise = rough * (3 + x*y*y) * 0.5
```

Comment:
Sorry, but what is `y`?

Reply:
As I'm already nitpicking there is 3 + x*y*y and in the code it's actually 3-x*y*y.

Reply:
I'll fix that.